README.md

```diff
@@ -16,7 +16,7 @@ De laatste versie is 1.3.1. Lees de [release notes](https://vng-realisatie.githu
 * Vraag een API-key voor toegang tot de Haal Centraal probeeromgeving aan bij de product owner [c.dingemanse@comites.nl](mailto:c.dingemanse@comites.nl)
 
 ## Heb je meer nodig?
-Gebruik de BRP bevragen API in combinatie met (een van de) andere BRP API’s:
+Gebruik de BRP Personen Bevragen API in combinatie met (een van de) andere BRP API’s:
 
 * [Historische BRP-gegevens bevragen](https://vng-realisatie.github.io/Haal-Centraal-BRP-historie-bevragen)
```

Comment:
Gaan we deze binnenkort dan ook hernoemen naar 'Historische BRP Personen Bevragen' of wordt daardoor te veel de associatie gelegd met historische personen waardoor het lijkt dat je daarover allerlei informatie kunt aanvragen, bijv. over Willem van Oranje?
Wellicht dat een betere naam dan 'BRP Personen Historie Bevragen' is.